release.sh

```diff
@@ -112,8 +114,17 @@ fi
 ./gradlew publish
 
 echo "Uploading executable artifacts to $S3_DIR"
+
 aws s3 cp --acl public-read sdk/bootstrap/bootstrap.zip $S3_DIR/bootstrap.zip 1>&2
+
 aws s3 cp --acl public-read sdk/executor/build/distributions/executor.zip $S3_DIR/executor.zip 1>&2
-aws s3 cp --acl public-read sdk/cli/dcos-service-cli-linux $S3_DIR/dcos-service-cli-linux 1>&2
-aws s3 cp --acl public-read sdk/cli/dcos-service-cli-darwin $S3_DIR/dcos-service-cli-darwin 1>&2
-aws s3 cp --acl public-read sdk/cli/dcos-service-cli.exe $S3_DIR/dcos-service-cli.exe 1>&2
+
+for cli_exe in $(ls sdk/cli/dcos-service-cli*); do
+    aws s3 cp --acl public-read $cli_exe $S3_DIR/$(basename $cli_exe) 1>&2
```

Comment:
Does `aws s3 cp` not support copying multiple files? An alternative may be `aws s3 sync` which does.